packages/synapse-react-client/src/components/FullWidthAlert/FullWidthAlert.tsx

```diff
@@ -116,50 +106,104 @@ function FullWidthAlert(props: FullWidthAlertProps) {
       }
     }
   }, [onClose, autoCloseAfterDelayInSeconds])
-  const additionalAlertVariantClass = variant ? `alert-${variant}` : ''
-  return (
+
+  const gridSize = 12
+  const buttonGridSize = 4
+  const hasButtons = primaryButtonConfig || secondaryButtonConfig
+  const buttonGridSizeMap = {
+    sm: hasButtons ? buttonGridSize + 1 : 0,
+    md: hasButtons ? buttonGridSize - 1 : 0,
+    lg: hasButtons ? buttonGridSize : 0,
+  }
+
+  const alert = (
     <Alert
-      variant={variant}
-      show={show}
-      dismissible={false}
-      transition={transition}
-      className={`FullWidthAlert bootstrap-4-backport ${
-        isGlobal ? 'global' : ''
-      } ${additionalAlertVariantClass}`}
+      severity={variantToSeverity(variant)}
+      sx={{
+        width: '100%',
+        '.MuiAlert-message': {
+          flexGrow: 1,
+        },
+      }}
+      className={`FullWidthAlert ${isGlobal && 'global'}`}
```

Comment:
Used for styling in `displayToast` and to adjust styling for SynapseNavDrawerisShowing